source/features/parse-backticks.tsx

```diff
@@ -15,18 +15,28 @@ function initGeneral(): void {
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span' // Pinned Issues
+		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'blame-commit-message a' // `isBlame`
 	])) {
 		parseBackticks(title);
 	}
 }
 
+export function parseBackticksOnNewsFeedItems(): void {
+	for (const title of select.all('.issues_labeled .text-gray-dark > a:not(.rgh-parse-backticks), .commits blockquote:not(.rgh-parse-backticks)')) {
+		parseBackticks(title);
+	}
+}
+
 // Highlight code in issue/PR titles on Dashboard page ("Recent activity" container)
 async function initDashboard(): Promise<void> {
 	await elementReady('.js-recent-activity-container', {stopOnDomReady: false});
 	for (const title of select.all('.js-recent-activity-container .text-bold')) {
 		parseBackticks(title);
 	}
+
+	await elementReady('.issues_labeled', {stopOnDomReady: false});
+	parseBackticksOnNewsFeedItems();
```

Comment:
Instead of this and the merge with `infinite-scroll`, I'd use a MutationObserver to call `parseDynamic`.

Also keep in mind that `{stopOnDomReady: false}` is unsafe: in this case for example it will run `document.querySelector` 60 times a second, forever, if my feed doesn't have that `.issues_labeled`. Currently, _my feed doesn't have it._

Reply:
MutationObserver what am I observing? Why would we not have the same issue? It will never disconnect if there are no issues?

Reply:
MutationObserver what am I observing? Why would we not have the same issue? It will never disconnect if there are no issues?